Product/Production/Gateway/PatientDiscovery_10/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/_10/gateway/ws/PatientDiscoverySpringContextTest.java

```diff
@@ -89,44 +106,43 @@
     @Test
     public void inbound() throws PRPAIN201305UV02Fault {
         assertNotNull(inboundPatientDiscoveryEndpoint);
-
         PRPAIN201305UV02 request = new PRPAIN201305UV02();
+        WebServiceContext context = new WebServiceContextImpl();
+        inboundPatientDiscoveryEndpoint.setContext(context);
         PRPAIN201306UV02 response = inboundPatientDiscoveryEndpoint.respondingGatewayPRPAIN201305UV02(request);
-
         assertNotNull(response);
     }
 
-   /* @Test
-    public void inboundFault() throws PatientDiscoveryException {
-        PRPAIN201305UV02 request = new PRPAIN201305UV02();
-
-        NhinPatientDiscovery inboundPDEndpoint = new NhinPatientDiscovery();
+    /* @Test
```

Comment:
If the test will compile, use the "Ignore" annotation instead of commenting it out.